### PR TITLE
fix: Add active flag to announcements

### PR DESCRIPTION
**Pull request by KristianSelnas**
Adds a property 'active' to announcements. This makes it easier to toggle announcements on and off without deleting them.

### CODE REVIEW

**Review by gorandalum — CHANGES_REQUESTED** [review]
Versions from master are a part of this PR. This should be cleaned up.

File: src/announcements/types.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import {FirebaseFirestoreTypes} from '@react-native-firebase/firestore';
 
 export type AnnouncementRaw = {
   id: string;
+  active: boolean;
```
Comment (gorandalum) [inline src/announcements/types.ts:6]: Do we need the field here since we use `active == true` in the query?

**Review by gorandalum — APPROVED** [review]
LGTM :+1:

File: src/announcements/types.ts (hunk 1 above)
Comment (KristianSelnas) [reply src/announcements/types.ts:6]: Belte og bukseseler, er det ikke det de kaller det?

**Pull request merged**